neo.UnitTests/UT_Transaction.cs

```diff
@@ -77,6 +78,90 @@ public void Size_Get()
             uut.Size.Should().Be(82);
         }
 
+        [TestMethod]
+        public void FeeIsMultiSigContract()
+        {
+            var store = TestBlockchain.GetStore();
+            var walletA = new NEP6Wallet("unit-test-A.json");
+            var walletB = new NEP6Wallet("unit-test-B.json");
+            var walletC = new NEP6Wallet("unit-test-C.json");
+            var snapshot = store.GetSnapshot();
+
+            using (var unlockA = walletA.Unlock("123"))
+            using (var unlockB = walletB.Unlock("123"))
+            using (var unlockC = walletC.Unlock("123"))
+            {
+                var a = walletA.CreateAccount();
+                var b = walletB.CreateAccount();
+
+                var multiSignContract = Contract.CreateMultiSigContract(1,
+                    new ECPoint[]
+                    {
+                        a.GetKey().PublicKey,
+                        b.GetKey().PublicKey
+                    }
+                    );
+
+                var acc = walletC.CreateAccount(multiSignContract);
```

Comment:
This created a watch-only address. You need to pass the second parameter `key` to it.

Reply:
works! :)